readme.md

```diff
@@ -1,7 +1,7 @@
 # Search bar for Airbnb
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://nataliyaverchenko.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://nataliyaverchenko.github.io/layout_search-bar-airbnb/report/html_report/)
```

Comment:
don't forget to add these links in pr description. readme displays in demo, try to fix it https://stackoverflow.com/questions/48919200/github-pages-only-showing-readme-file

Reply:
Please help me decide, I tried to change it and I don't know why it didn't work
![image](https://user-images.githubusercontent.com/17496682/208269574-3fa60a9b-8c58-47ec-8bbb-e201e93dc6e1.png)

Reply:
I have no idea how to fix this but the links don't work

Reply:
![image](https://user-images.githubusercontent.com/17496682/208270025-c423cd5d-cfae-4da0-9d15-13ff30b83119.png)

I follow this instruction and don't understand why the links don't work and what exactly needs to be done

Reply:
> don't forget to add these links in pr description. readme displays in demo, try to fix it https://stackoverflow.com/questions/48919200/github-pages-only-showing-readme-file

I don't understand what you mean ((